src/scss/skin-modern/_variables.scss

```diff
@@ -20,3 +20,5 @@ $subtitle-text-border-color: #000 !default;
 
 $animation-duration: .3s !default;
 $animation-duration-short: $animation-duration / 2 !default;
+
+$seekbar-height: 5px;
```

Comment:
Move this into `_seekbar.scss`, it's not a global setting

Reply:
i need it also in the `_volumeslider.scss` so either declaring it two times or moving somewhere else. Is there a way to 'inherit' it?

Reply:
Declare it at the top of `_seekbar.scss` (below the imports) and add an `@import '_seekbar';` to `_volumeslider.scss`